src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/SupportApiServiceImpl.scala

```diff
@@ -0,0 +1,196 @@
+package it.pagopa.interop.backendforfrontend.api.impl
+
+import akka.http.scaladsl.marshalling.ToEntityMarshaller
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives.{onComplete, redirect}
+import com.typesafe.scalalogging.{Logger, LoggerTakingImplicit}
+import it.pagopa.interop.commons.jwt.service.SessionTokenGenerator
+import it.pagopa.interop.commons.jwt._
+import it.pagopa.interop.commons.signer.model.SignatureAlgorithm
+import it.pagopa.interop.commons.logging.{CanLogContextFields, ContextFieldsToLog}
+import it.pagopa.interop.backendforfrontend.common.system.ApplicationConfiguration
+import it.pagopa.interop.backendforfrontend.api.SupportApiService
+import it.pagopa.interop.backendforfrontend.service.TenantProcessService
+import it.pagopa.interop.backendforfrontend.model._
+import it.pagopa.interop.tenantprocess.client.{model => TenantProcess}
+import it.pagopa.interop.backendforfrontend.error.BFFErrors._
+import it.pagopa.interop.backendforfrontend.error.Handlers.handleError
+import it.pagopa.interop.commons.utils.service.OffsetDateTimeSupplier
+import it.pagopa.interop.commons.utils.TypeConversions._
+import it.pagopa.interop.commons.utils._
+import javax.xml.parsers.DocumentBuilderFactory
+import org.w3c.dom
+import org.opensaml.saml2.core.Response
+import org.opensaml.security.SAMLSignatureProfileValidator
+import org.opensaml.xml.validation.ValidationException
+import org.opensaml.xml.{XMLObject, Configuration}
+import org.opensaml.DefaultBootstrap
+import org.opensaml.xml.io.{Unmarshaller, UnmarshallerFactory}
+
+import cats.syntax.all._
+import java.time.{OffsetDateTime, Instant, ZoneOffset}
+import java.io.ByteArrayInputStream
+import scala.jdk.CollectionConverters._
+import scala.concurrent.{Future, ExecutionContext}
+import scala.util.Success
+
+final case class SupportApiServiceImpl(
+  sessionTokenGenerator: SessionTokenGenerator,
+  tenantProcessService: TenantProcessService,
+  offsetDateTimeSupplier: OffsetDateTimeSupplier
+)(implicit ec: ExecutionContext)
+    extends SupportApiService {
+
+  private implicit val logger: LoggerTakingImplicit[ContextFieldsToLog] =
+    Logger.takingImplicit[ContextFieldsToLog](this.getClass)
+
+  val SUPPORT_LEVELS: Seq[String] = Seq("L2", "L3")
+  val SUPPORT_LEVEL_NAME: String  = "supportLevel"
+  val OPERATOR: String            = "OPERATOR"
```

Comment:
```suggestion
  val PARTY_ROLE_OPERATOR: String            = "OPERATOR"
```

I don't remember, is uppercase correct?

Reply:
In the story is in uppercase style

Reply:
I confirm the uppercase style.
@galales the concept party is destinated to disappear.
I suggest using `SELFCARE_OPERATOR_ROLE`.